### DISCUSSION TITLE
$co und $is im Zertifikats pdf

### DISCUSSION THREAD

**Discussion by marcusthierfelder** [Digitaler-Impfnachweis/certification-apis#134] [General]
Hallo,

seit heute erscheint im pdf unter Mitgliedsstaat und Austeller nur noch die Platzhalter. Ändern oder weglassen der Daten im JSON führt zu keiner Änderung. Gestern ging es noch.

[filename.pdf](https://github.com/Digitaler-Impfnachweis/certification-apis/files/6699532/filename.pdf)

**Comment by thinkberg** [comment]
Ist gerade eben repariert word, da hatte sich ein Bug eingeschlichen :(

**Reply by marcusthierfelder** [reply]
Vielen Dank für die schnelle Hilfe, wir haben es schon gesehen.

**Reply by thinkberg** [reply]
So schnell war ich morgens noch nie aus dem Bett ...
